src/utils/identity.js

```diff
@@ -0,0 +1,3 @@
+export default function identity(value) {
+  return value;
+}
```

Comment:
It's probably best to put these all in a single file and export them individually that way you avoid the overhead of including each file, I think it adds ~3 lines and a bunch of bytes for every file you include.

This also has the benefit of keeping all the utils together.